spec/lib/action_kit/helper_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+describe ActionKit::Helper do
+  before do
+    ENV['AK_PASSWORD'] = 'ak_password'
+    ENV['AK_USERNAME'] = 'ak_username'
```

Comment:
I'm controlling the environment in which my specs are run, which is perfectly ordinary. I'm not privy to what the weird inconsistencies were, but I think we should try and understand what the problem was rather than shy away from using `ENV` in our spec contexts.

I'll return their values to `nil` in the `after` block.

@Tuuleh the specs below the hook depend on the values I've set.

I'll reset in the